docs/ui.md

```diff
@@ -17,4 +17,28 @@ You can also use other web servers to serve the contents of the www/app director
 ### Running remotely
 When Kubernetes is deployed remotely, the UI is deployed as a cluster addon. To access it, visit `/ui`, which redirects to `/api/v1/proxy/namespaces/default/services/kube-ui/#/dashboard/`, on your master server.
 
+## Using the UI
```

Comment:
Also, I would get rid of "You can also use other web servers to serve the contents of the www/app directory, as described [here](../www/README.md#serving-the-app-during-development)." -- I think this is confusing (web servers other than what? why would I want to use another web server?), and anyway people can find this info if they follow the link you already provide to that doc.